tests/test_forward_selection.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.datasets import make_friedman1
+from sklearn.linear_model import LinearRegression
+
+from feature_selection import forward_selection
+
+def scorer(X, y):
+    """
+    Sample custom scorer that fits a model and returns
+    an appropriate score for the feature selection problem.
+    Parameters
+    ----------
+    X : array-like of shape (n_samples, n_features)
+        Test samples
+    y : array-like of shape (n_samples,) or (n_samples, n_outputs)
+        True values for X, used for training
+    Returns
+    -------
+    Error of scorer
+    """
+    lr = LinearRegression().fit(X, y)
+    return 1 - lr.score(X, y)
+
+
+def test_forward_selection():
```

Comment:
Look at other PRs to get some more ideas for testing.

Reply:
ok, thanks